src/styles/credential.css

```diff
@@ -55,3 +55,16 @@
     display: flex;
     justify-content: center;
 }
+
+.add-credential-api-doc{
+    font-size: 6px;
+    font-weight: bold;
+    float: right;
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    display: inline;
+    padding: 0;
+    margin-top: -27px;
+    margin-right: 680px;
```

Comment:
we can move it we more to the right. We need to be similar to the service ordering page.
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/f42d9dbd-0195-497f-8625-4a172b084329)